documentation/src/main/antora/antora-playbook.yml

```diff
@@ -0,0 +1,18 @@
+# partial playbook just for this project to be able to compile documentation without pushing it to GitHub
+# usage: antora generate antora-playbook.yml
+site:
+    title: SmallRye Reactive Messaging Documentation
+    start_page: smallrye-reactive-messaging::index.adoc
+content:
+    sources:
+        - url: ./
```

Comment:
Antora needs to run from a git repo, but if you want to spare yourself the need to do `git init` you can change this to

```
content:
    sources:
        - url: ../../..
          start_path: documentation/target/antora
          branches: HEAD                    
``` 

and it will reuse the existing git repo